dpks/quant_matrix.py

```diff
@@ -531,6 +533,55 @@ def impute(self, method: str, **kwargs: int) -> QuantMatrix:
 
         return self
 
+    def plot(
+        self,
+        plot_type: str,
+        save: bool = False,
+        fig=None,
+        ax=None,
+        **kwargs: Union[
+            np.ndarray,
+            int,
+            list,
+            str,
+        ],
+    ) -> matplotlib.pyplot.Figure:
```

Comment:
This return statement isn't valid anymore since a tuple (fig, ax) is returned. Should change to ```tuple[matplotlib.figure.Figure, matplotlib.axes.Axes]```, or the corresponding objects